kobo/apps/help/models.py

```diff
@@ -45,7 +45,7 @@ class InAppMessage(AbstractMarkdownxModel):
     last_editor = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
     # We do not want to use a generic foreign key or tightly couple this model
     # with another one, so we use JSONField to store related object name and pk
-    generic_related_objects = models.JSONField(default=dict)
+    generic_related_objects = models.JSONField(default=dict, blank=True)
```

Comment:
It looks like this should get a migration 👍

Reply:
Should not be there at the first glance :-(